src/Resources/views/BlockAdmin/compose_preview.html.twig

```diff
@@ -7,21 +7,30 @@
     <a class="page-composer__container__child__edit"
        href="{{ blockAdmin.generateUrl('edit', { 'id': child.id, 'composer': true }) }}"
     >
-        {% set service = attribute(blockServices, child.type) %}
-        {% if service.metadata is defined %}
-            {% set metadata = service.metadata %}
+        {% set service = attribute(blockServices, child.type) ?? null %}
+
+        {% if service is not null %}
+            {% set blockTitle = child.name|default(service.metadata.title)|trans({}, service.metadata.domain|default('SonataPageBundle')) %}
+            {% set blockImage = service.metadata.image ? asset(service.metadata.image) : null %}
+            {% set blockClass = service.metadata.option('class') %}
+            {% set metadataTitle = service.metadata.title|trans({}, service.metadata.domain|default('SonataPageBundle')) %}
         {% else %}
-            {% set metadata = service.blockMetadata %}
+            {% set blockTitle = child.name|trans({}, 'SonataPageBundle') %}
```

Comment:
On the previous code child.name was translated, but that does not make a lot of sense to me. Am I right? @VincentLanglet @eerison @JeromeEngelnAdeliom

Reply:
Translate child.name does not make sense! because it is a dynamic data from db